versions.props

```diff
@@ -26,6 +26,7 @@
     <MySqlV8>8.0.23</MySqlV8>
     <NpgsqlVersion>4.0.0</NpgsqlVersion>
     <SqlClientVersion>4.8.1</SqlClientVersion>
+    <MicrosoftSqlClientVersion>5.0.*</MicrosoftSqlClientVersion>
```

Comment:
This should probably be a static version rather than floating

Reply:
Right, thanks!